earth_enterprise/src/fusion/autoingest/JsonUtilsV1.cpp

```diff
@@ -0,0 +1,337 @@
+// Copyright 2017 Google Inc. Copyright 2018 the Open GEE Contributors.
```

Comment:
Why was this done as a separate copy of the utility instead of just adding a parameter to the JsonObject method inside JsonUtils.cpp/h? Can you refactor it like that?